docs/source/nemo_tools/text_denormalization.rst

```diff
@@ -0,0 +1,13 @@
+Text Denormalization
+====================
+
+Inverse text normalization (ITN), also called denormalization, is a part of the Automatic Speech Recognition (ASR) post-processing pipeline.
+ITN is the task of converting the raw spoken output of the ASR model into its written form to improve the text readability.
```

Comment:
Could you do this in new sentence with --> arrows so the input -> output stands more out?